teams/wg-async.toml

```diff
@@ -38,7 +38,7 @@ alumni = [
 orgs = ["rust-lang", "rust-lang-nursery"]
 
 [website]
-name = "Async Foundations working group"
+name = "Async working group"
 description = "Pursuing core language and library support for async-await"
```

Comment:
nit: I think `async/await` is a more standard spelling? The current spelling is probably fine though.

Reply:
I'm going to merge, but feel free to submit a change to this in a new PR